src/backoffice/core/property-editors/uis/label/property-editor-ui-label.element.ts

```diff
@@ -9,16 +9,17 @@ import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
  */
 @customElement('umb-property-editor-ui-label')
 export class UmbPropertyEditorUILabelElement extends UmbLitElement implements UmbPropertyEditorExtensionElement {
-
-
 	@property()
 	value = '';
 
+	@property()
+	description = '';
+
 	@property({ type: Array, attribute: false })
-	public config = [];
+	config = [];
 
 	render() {
-		return html`<div>umb-property-editor-ui-label</div>`;
+		return html`${this.value}`;
```

Comment:
This value could contain anything such as custom HTML and scripts. We should probably test it to see if it prints out HTML correctly or if it is escaped. I know that some extensions save HTML directly in a label. We should probably also figure out if we can scrape it a bit to remove dangerous characters by running it through an HTML parser such as [sanitize-html](https://www.npmjs.com/package/sanitize-html)